test/archive/har.d

```diff
@@ -0,0 +1,176 @@
+module archive.har;
```

Comment:
I think it would be helpful to add a documentation header to this file summarizing the HAR format. Perhaps some cut and paste from the readme at https://github.com/marler8997/har.  A link to that repository would also be helpful to know where it came from to add to the maintainer's historical and contextual understanding.

Reply:
ok